ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -14,13 +14,16 @@ public typealias MSPillButtonStyle = PillButtonStyle
 public enum PillButtonStyle: Int {
     case outline
     case filled
+    case transparent
 
     func backgroundColor(for window: UIWindow) -> UIColor {
         switch self {
         case .outline:
             return Colors.PillButton.Outline.background
         case .filled:
             return UIColor(light: Colors.primaryShade10(for: window), dark: Colors.PillButton.Outline.background)
+        case .transparent:
+            return .clear
```

Comment:
We'll probably want to add this to the Colors.Swift file, even if it is just `.clear`

Reply:
It is not a good practice to add variables provided by Apple in a global scope, because that will increase the project/framework size, as well as the layer complexity.